test/runtests.jl

```diff
@@ -101,6 +104,10 @@ end
     @test isapprox(loglikelihood(glm_model), -4353.946729075838)
     @test isapprox(nullloglikelihood(lm_model), -4984.892139711452)
     @test isapprox(mean(residuals(lm_model)), -5.412966629787718) 
+
+    # this should be elementwise true (which is a stronger condition than
+    # vectors being approximately equal) the so we test elementwise
+    @test all(residuals(glm_model) .≈ residuals(lm_model))
```

Comment:
```suggestion
    @test all(residuals(glm_model, type = :working) .≈ residuals(lm_model))
```
the `lm residuals` should be compared with `glm residuals with working` type while comparing with the corresponding `glm`. Now the test case should be passed as I have checked locally.